scripts/prepare-unity-mobile.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+### This script is used to setup Improbable's internal build machines.
+### If you don't work at Improbable, this may be interesting as a guide to what software versions we use for our
+### automation, but not much more than that.
+
+set -e -u -x -o pipefail
```

Comment:
More general question - how this code is intended to be used?

Reply:
In the GDK scripts we clone the `gdk-for-unity-shared-ci` repo and call these scripts from there.

Reply:
Can we (and do we want to) alter the script so that it runs on initial project launch? If we can user can skip the process of setting it up manually

Reply:
This requires the users to set up environment variables manually which I feel is more troublesome than simply adding it with the Unity GUI.

Reply:
Fair enough, let's keep it limited to CI